stacks-common/src/address/c32.rs

```diff
@@ -18,12 +18,40 @@ use super::Error;
 
 use sha2::Digest;
 use sha2::Sha256;
-
-const C32_CHARACTERS: &str = "0123456789ABCDEFGHJKMNPQRSTVWXYZ";
+use std::convert::TryFrom;
+
+const C32_CHARACTERS: &[u8; 32] = b"0123456789ABCDEFGHJKMNPQRSTVWXYZ";
+
+/// C32 chars as an array, indexed by their ASCII code for O(1) lookups.
+/// Supports lookups by uppercase, lowercase, and special (i.e. `O, L, I`) chars.
+/// Table can be generated with:
```

Comment:
Can you add a comment to this docstring to indicate that this table will encode `O`, 'o', and `0` as `0`, and `1, `l`, `L`, 'l', `I`, and `i` as `1`?